plot.py

```diff
@@ -1,19 +1,31 @@
+import argparse
+import glob
 import json
 import matplotlib.pyplot as plt
-import glob
 import re
 
+parser = argparse.ArgumentParser(description='Plot results from compare.py')
+parser.add_argument('-em', '--eval-metric',
+                    dest='metric',
+                    help='Plot a specific metric',
+                    default='rmse')
+options = vars(parser.parse_args())
+
+metric = options.get('metric')
 
 fig, ax = plt.subplots(1, 1)
 
 for filename in glob.glob('results/*.json'):
     title = re.match('results/(.*).json', filename).group(1)
+    if title[:3] == "svd":
+        continue
```

Comment:
hm, why?

Reply:
Leftovers from debugging, used to prevent a crash before better safeguards

Reply:
In this case maybe add a comment